inst/pages/12_quality_control.qmd

```diff
@@ -301,7 +301,7 @@ plotColData(tse,"sum","SampleType", colour_by = "SampleType") +
 ```
 
 In addition, data can be rarefied with
-[subsampleCounts](https://microbiome.github.io/mia/reference/subsampleCounts.html),
+[rarefyAssay](https://microbiome.github.io/mia/reference/rarefyAssay.html),
```

Comment:
I would also add that "This remains controversial, however, and strategies to mitigate the information loss in rarefaction have been proposed [@citations]."

Add these papers in the bibtext file and cite:
https://journals.asm.org/doi/10.1128/msphere.00354-23
https://journals.asm.org/doi/10.1128/msphere.00355-23